lib/mumukit/platform/organization/profile.rb

```diff
@@ -19,4 +23,16 @@ def locale_json
   def logo_url
     @logo_url ||= 'https://mumuki.io/logo-alt-large.png'
   end
+
+  def banner_url
+    @banner_url ||= logo_url
+  end
+
+  def favicon_url
+    @favicon_url ||= '/favicon.ico'
+  end
+
+  def open_graph_image_url
```

Comment:
Although those changes are very simple, I'd add a test for the getters with defaults.